### DISCUSSION TITLE
I would like to know Apple's view on mobile apps using Web3 Auth.

### DISCUSSION THREAD

**Discussion by HidetakaKo** [Web3Auth/Web3Auth#675] [Help]
We are considering using Web3 Auth to implement an email address and Apple Sign in, as we need a way to log in that does not rely solely on Connect Wallet in order to pass Apple's review. So I would like to know Apple's view on mobile apps using Web3 Auth.

**Comment by Haris565** [comment]
We deployed our App in app store and it got rejected three time because they were not able to Signin with Apple but the issue is not reproduced after testing on many devices from our side

**Reply by BboyStatix** [reply]
@Haris565 do you by any chance have any additional info or console logs that could help us debug the issue?
